src/Greenshot/Languages/language-de-DE.xml

```diff
@@ -57,8 +57,8 @@ schnell zu finden. Vielen Dank :)</resource>
 		<resource name="contextmenu_donate">Greenshot unterstützen</resource>
 		<resource name="contextmenu_exit">Schließen</resource>
 		<resource name="contextmenu_help">Hilfe</resource>
-		<resource name="contextmenu_openfile">Datei öffnen</resource>
-		<resource name="contextmenu_openrecentcapture">Letzten Screenshot im Explorer anzeigen</resource>
+		<resource name="contextmenu_openfile">Datei öffnen...</resource>
+		<resource name="contextmenu_openrecentcapture">Ordner des letzten Screenshots im Explorer öffnen</resource>

```

Comment:
I suggest to keep the original message because I think it is shorter, easier to read and also more concise: the feature does not only open the folder, it also selects the file.

```suggestion
		<resource name="contextmenu_openrecentcapture">Letzten Screenshot im Explorer anzeigen</resource>
```

Reply:
I agree.